e2e-tests/functions

```diff
@@ -11,12 +11,12 @@ perconalab/percona-server-mongodb-operator:main-mongod5.0
 
 perconalab/percona-server-mongodb-operator:main-mongod6.0
 perconalab/percona-server-mongodb-operator:main-mongod7.0'}
-IMAGE_BACKUP=${IMAGE_BACKUP:-"perconalab/percona-server-mongodb-operator:main-backup"}
+IMAGE_BACKUP=${IMAGE_BACKUP:-"percona/percona-backup-mongodb:2.5.0"}
```

Comment:
there is version 2.6.0 
could we use it?

Reply:
Hi, 
There is an issue with 2.6.0 https://perconacorp.slack.com/archives/GDW1BCK6H/p1725952692706359 . So I am reverting to 2.5.0 and we will use 2.6.0 once fixed.